### ISSUE TITLE
[build system] missing checks on libhd

### ISSUE THREAD

**Issue by abucodonosor** [manjaro/mhwd#33]
Hello,
without libhd intstalled mhwd will error out ..

...
https://github.com/manjaro/mhwd/blob/master/libmhwd/CMakeLists.txt#L16
https://github.com/manjaro/mhwd/blob/master/libmhwd/CMakeLists.txt#L20
...

but nothing checks to see the lib is around..

**Comment by philmmanjaro** [comment]
I don't know your problem here. **mhwd** has a dependency on **hwinfo** which ships **libhd**.

**Comment by abucodonosor** [comment]
Yes your PKGBUILD has hwinfo in depends=()  but when you try to build it from sources
without hwinfo installed you'll get an compile error.
